modules/core/src/main/scala/gem/math/Quantities.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package math
+
+import libra.{ QuantityOf, Unit }
+import spire.math.UInt
+
+/**
+ * Typeclass that can create a Quantity from a UInt
+ * Useful to create generic encoders
+ */
+trait QuantityOfFromUInt[A, B <: Unit[A]] {
```

Comment:
This is a typeclass to build `QuantityOf`. I'm not sure how sound this is but it is useful to write generic encoders